### DISCUSSION TITLE
ability to save surfaces object only

### DISCUSSION THREAD

**Discussion by xyont** [Macad3D/Macad3D#47] [General]
all model build results is in solid object, for some purpose needed to save of surface only. 

is this possible to provide such as options?

**Comment by dipts** [comment]
I think the best way to do this is to use a script. With it you can select the faces to be exported according to any criteria. Or make them selectable via a tool. Examples for the creation of scripts can be found in the [user guide](https://macad3d.net/userguide/html/8bd40f7f-0687-46fe-ba84-2faf6bc91571.htm).
